publish/samplesite/scripts/controls/plugins.ts

```diff
@@ -16,6 +18,7 @@ export default interface Plugins {
     eventView: EventViewPlugin;
     api: ApiPlaygroundPlugin;
     customReplace: CustomReplacePlugin,
+    picker: PickerPlugin,
```

Comment:
Please move customReplace and picker plugin declarations [here](https://github.com/Microsoft/roosterjs/blob/00553d8646c059502125f3b870566e9d2b1a1f2f/publish/samplesite/scripts/controls/editor/Editor.tsx#L76), so that they are configurable from side pane.

And suggest adding them into the "Editor Options" side pane, and add sample code for it under "Typescript Code" section.